dashboard/app/jobs/evaluate_rubric_job.rb

```diff
@@ -422,7 +441,8 @@ def validate_ai_config_for_lesson(lesson_s3_name, code)
           learning_goal_id: lg.id,
           rubric_ai_evaluation_id: rubric_ai_evaluation.id,
           understanding: understanding_s_to_i(label),
-          ai_confidence: ai_evaluation['Confidence'],
+          ai_confidence: ai_evaluation['Confidence-Pass-Fail'],
+          ai_confidence_exact_match: ai_evaluation['Confidence-Exact-Match'],
```

Comment:
nit: is this key casing/format consistent with other keys? I see 'Key Concept' above

Reply:
good point! fixed